lib/entities/wallet_list_order_types.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:cake_wallet/generated/i18n.dart';
+
+enum WalletListOrderType {
+  CreationDate,
+  Alphabetical,
+  GroupByType,
+  Custom;
+
+  @override
+  String toString() {
+    switch (this) {
+      case WalletListOrderType.CreationDate:
+        return S.current.creation_date;
+      case WalletListOrderType.Alphabetical:
+        return S.current.alphabetical;
+      case WalletListOrderType.GroupByType:
+        return S.current.group_by_type;
+      case WalletListOrderType.Custom:
+        return S.current.custom;
```

Comment:
let's change this to be `Custom (Hold and Drag)` or if you have a better wording that elaborates the custom sorting mechanism